staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -725,6 +725,8 @@ message Capability {
     repeated string add_capabilities = 1;
     // List of capabilities to drop.
     repeated string drop_capabilities = 2;
+    // List of ambient capabilities to add.
```

Comment:
This should document the interactions with `add_capabilities` and `drop_capabilities`. In other words, what happens if a capability is added to multiple lists? Or is that an invalid request?

Reply:
Let's add a link to https://man7.org/linux/man-pages/man7/capabilities.7.html?

Reply:
Done.